bigip/resource_bigip_ltm_virtual_server.go

```diff
@@ -45,6 +45,13 @@ func resourceBigipLtmVirtualServer() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"state": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "enabled",
```

Comment:
I don't think this is needed, an explicit `Default` should only be used when the managed device doesn't provide it, however Bigip enables all Virtual Servers by Default and the Read method after create will retrieve this and update the state accordingly.

Reply:
if i dont specify Default State value, there are terraform state related issues seen.
ex: for the first terraform apply it is Created VS with state enabled and state will be set to `enabled`. but for second terraform apply with out any change it is assuming there is change in state from `enabled` to `null`

Reply:
```
if vs.Enabled {
		d.Set("state", "enabled")
	}
	if vs.Disabled {
		d.Set("state", "disabled")
	}
```
above logic not solving issue...are u seeing any issue with taking default value of state as `enabled` ?

Reply:
@dannyk81 
Still we are running into one Issue.
Lets say we created VS with state given as `disabled`  and now i want to remove state variable from .tf file and give terraform apply will not show any state change. here i am expecting state should be changed `enabled`.